python-project/mortar/tesseract.py

```diff
@@ -1,5 +1,6 @@
 """
-This module provides an interface for performing OCR operations using Tesseract.
```

Comment:
What's the reason for this formatting change?

Reply:
1 character too long, failed flake8 linting